hyprctl/Strings.hpp

```diff
@@ -81,6 +82,14 @@ const std::string_view HYPRPAPER_HELP = R"#(usage: hyprctl [flags] hyprpaper <re
 flags:
     See 'hyprctl --help')#";
 
+const std::string_view HYPRSUNSET_HELP = R"#(usage: hyprctl [flags] hyprsunset <request>
+
+requests:
+    TODO!!!
```

Comment:
todooooo, either remove or fill out. I'd just recommend removing, it should be in the wiki